docs/install.md

```diff
@@ -131,7 +143,7 @@ Important: You need to run `pip uninstall mmcv` first if you have mmcv installed
 pip install mmdet
```

Comment:
To install the newest mmdet, `pip install git+https://github.com/open-mmlab/mmdetection.git` is a better choice.

Reply:
Addressed in https://github.com/open-mmlab/mmhuman3d/pull/237/commits/317ecc4ec07cd447693255294d09cabe8ccbe2bc.